packages/server/graphql/mutations/helpers/removeTeamMember.ts

```diff
@@ -136,11 +138,20 @@ const removeTeamMember = async (
 
   const archivedTasks = await archiveTasksForDB(integratedTasksToArchive)
   const archivedTaskIds = archivedTasks.map(({id}) => id)
+  const agendaItemIds = await r
+    .table('AgendaItem')
+    .getAll(teamId, {index: 'teamId'})
+    .filter((row) => row('teamMemberId').eq(teamMemberId))
+    .getField('id')
+    .coerceTo('array')
+    .distinct()
```

Comment:
no need for `coerceTo` or `distinct` since the coerceTo is implied & they're UIDs